pkg/mcs/resourcemanager/server/apis/v1/api.go

```diff
@@ -159,10 +161,16 @@ func (s *Service) putResourceGroup(c *gin.Context) {
 //	@Failure	404		    {string}	error
 //	@Param		name	    path		string	true	"groupName"
 //	@Param		with_stats	query		bool	false	"whether to return statistics data."
+//	@Param		keyspace_id	query		uint32	false	"keyspace ID, if not set, it will be treated as null keyspace ID."
```

Comment:
Since the API is intended for external use rather than internal, we should use the keyspace name as the parameter. You might want to wait until #9354 is merged.